### PR TITLE
Show the lightbulb when there are available refactorings at the current location.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Previously, we never brought up the lightbulb when there were refactorings available.  Users had to just know they existed, and would have to hit ```ctrl-,``` explicitly.   We did an A/B test where we just proffered the lightbulb when refactorings were available, and we saw a nice increase in people using these features.  No data indicated a negative sentiment from people.  So we're just going to go with this approach from now on.

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
retest microbuild_prtest please

**Comment by CyrusNajmabadi** [comment]
Tagging @dotnet/roslyn-ide @kuhlenh

File: src/EditorFeatures/Core/Implementation/CodeFixes/ICodeFixService.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ namespace Microsoft.CodeAnalysis.CodeFixes
 {
     internal interface ICodeFixService
     {
-        Task<FirstDiagnosticResult> GetFirstDiagnosticWithFixAsync(Document document, TextSpan textSpan, bool considerSuppressionFixes, CancellationToken cancellationToken);
+        Task<FirstDiagnosticResult> GetFirstDiagnosticWithFixAsync(Document document, TextSpan textSpan, CancellationToken cancellationToken);
```
Comment (tmeschter) [inline src/EditorFeatures/Core/Implementation/CodeFixes/ICodeFixService.cs]: So basically the `considerSuppressionFixes` parameter controlled whether or not we showed refactorings?

Comment (tmeschter) [inline src/EditorFeatures/Core/Implementation/CodeFixes/ICodeFixService.cs]: I ask because that's not _at all_ what I would think that means.

**Pull request merged**